src/main/resources/staff/templates/document/countyCourtJudgmentDetails.html

```diff
@@ -3,12 +3,36 @@
   <meta charset="UTF-8"/>
   <title>{{ ccj.claim.referenceNumber }}</title>
   <style>
+    @page :first {
+      @top-right {
+        content: "";
+      }
+    }
+
     @page {
       size: 8.5in 11in;
       margin: 10%;
 
+      @top-right {
+        content: "Claim number: {{ ccj.claim.referenceNumber }}";
+        font-size: 18px;
+        font-family: "Open Sans";
+      }
+
       @bottom-left {
-        content: "OCON205A Request for judgment by default (07.17)";
+        content: "OCON225";
```

Comment:
Prototype linked to the story has `N225`, is `OCON225` correct?

Reply:
Art asked me to update content